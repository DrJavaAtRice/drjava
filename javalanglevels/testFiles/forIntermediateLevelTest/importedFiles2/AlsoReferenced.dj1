package forIntermediateLevelTest.importedFiles2;

public class AlsoReferenced {
}