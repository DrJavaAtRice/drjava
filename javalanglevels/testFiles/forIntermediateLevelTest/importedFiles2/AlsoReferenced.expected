package forIntermediateLevelTest.importedFiles2;

public class AlsoReferenced {

  /** This method is automatically generated by the Language Level Converter. */
  public AlsoReferenced() {
    super();
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      AlsoReferenced cast = ((AlsoReferenced) o);
      return true;
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode();
  }
}